### ISSUE TITLE
Could not upload files iRODS 4.3 + metalnx 2.6

### ISSUE THREAD

**Issue by hanytran** [irods-contrib/metalnx-web#323]
Hi,
thanks for your time checking my previous issues due to old versions.

I decided to run iRODS 4.3 outside docker-compose with Metalnx 2.6.0 but no file can be uploaded (even with 2.5 or 2.4 no luck).
However, folders can be created.

iCommand is working fine.

Do I need to adjust anything in the metalnx settings?

![Screenshot from 2022-09-09 10-51-04](https://user-images.githubusercontent.com/36070041/189311495-de3c30f3-d6a4-48be-8a37-aaba71f9ae29.png)

**Comment by korydraughn** [comment]
Please check the log files for Metalnx and the iRODS server (/var/log/irods/irods.log). The logs should provide hints as to what is happening.

**Comment by hanytran** [comment]
Just wonder why there is nothing in it.

![Screenshot from 2022-09-09 17-02-07](https://user-images.githubusercontent.com/36070041/189381270-8b35529b-0abd-4386-946c-988dc57f53b8.png)

**Comment by hanytran** [comment]
In Metalnx, only one WARN, the rest is INFO
![Screenshot from 2022-09-09 17-08-51](https://user-images.githubusercontent.com/36070041/189383055-054c3177-7d6c-4143-a91c-89c9eaf19052.png)

**Comment by korydraughn** [comment]
That is surprising. I'll give it a try.

In the meantime, how did you deploy iRODS 4.3 and Metalnx?

**Comment by hanytran** [comment]
In a VM ubuntu 20.04, 
- iRODS 4.3 was installed with this package https://packages.irods.org/, guides from here https://github.com/irods/irods_training/blob/main/beginner/irods_beginner_training_2022.pdf
- Metalnx 2.6 with docker compose from here https://github.com/irods-contrib/metalnx-web
- in metalnx.properties (in metalnx-configuration), irods.host is replaced with the hostname of the VM
That's it. And file upload was failed without logging tracked.

**Comment by hanytran** [comment]
P/S: same behavior with iRODS 4.2.11 + Metalnx 2.6 (or 2.5 or 2.4), however this setup I used iRODS docker image and all of them are in docker compose

**Comment by korydraughn** [comment]
And the icommands work for the ubuntu 20 VM?

**Comment by hanytran** [comment]
Yes, iCommands is working fine.

**Comment by korydraughn** [comment]
When you attempt to upload a file, what messages appear in the Metalnx log output?

Feel free to post the log output here.

**Comment by hanytran** [comment]
I'll be back on Monday and let you know.

**Comment by korydraughn** [comment]
I just tried uploading a file using Metalnx 2.6.0 to an iRODS 4.3.0 server and it worked.

There may be some external factor that is causing the behavior you're seeing.

Seeing the log output should help track down what is happening.

**Comment by hanytran** [comment]
You're using the same deployment? 
It's just so weird to my installations.

**Comment by hanytran** [comment]
Or which OS/setup you're using? I will ask my organization for that and try.

**Comment by korydraughn** [comment]
I've been running iRODS 4.3.0 using the testing environment (docker/ubuntu 20.04/postgresql).

I've been running metalnx 2.6.0 in a separate docker container (via the docker-compose project that ships with it).

My host OS is Ubuntu 20.04.

**Comment by hanytran** [comment]
> I've been running iRODS 4.3.0 using the testing environment (docker/ubuntu 20.04/postgresql).
> 
> I've been running metalnx 2.6.0 in a separate docker container (via the docker-compose project that ships with it).
> 
> My host OS is Ubuntu 20.04.

I picked up another VM and installed like this. Unfortunately I didn't notice that was ubuntu 22.04 and it just failed at the very first step of installing iRODS server. Will give it another try tomorrow in ubuntu 20.04.
The one installed last week was removed, so I could not send you the error log.

![Screenshot from 2022-09-12 11-03-02](https://user-images.githubusercontent.com/36070041/189688797-a647d9b4-93ce-4c2c-b408-793e61193b0f.png)
